src/tokenizer.js

```diff
@@ -1387,6 +1433,10 @@ export default class Tokenizer {
         return this.scanStringLiteral();
       }
 
+      if (charCode === 0x60) {
```

Comment:
Elsewhere we add a comment indicating which character this is and we use four hexits.